terraform/ec2/amis.tf

```diff
@@ -380,41 +399,41 @@ locals {
   arch = var.amis[var.testing_ami]["arch"] == "amd64" ? "x86_64" : var.amis[var.testing_ami]["arch"]
 }
 
-data "aws_ami" "selected" {
-  most_recent = true
+# data "aws_ami" "selected" {
+#   most_recent = true
 
-  filter {
-    name = "name"
-    values = [
-    var.amis[var.testing_ami]["ami_search_pattern"]]
-  }
+#   filter {
+#     name = "name"
+#     values = [
+#     var.amis[var.testing_ami]["ami_search_pattern"]]
+#   }
 
-  filter {
-    name   = "architecture"
-    values = [local.arch]
-  }
+#   filter {
+#     name   = "architecture"
+#     values = [local.arch]
+#   }
 
-  filter {
-    name = "state"
-    values = [
-    "available"]
-  }
+#   filter {
+#     name = "state"
+#     values = [
+#     "available"]
+#   }
 
-  # Only apply product code filter for some AMI, e.g. debian10
-  # https://www.terraform.io/docs/language/expressions/dynamic-blocks.html
-  dynamic "filter" {
-    for_each = var.amis[var.testing_ami]["ami_product_code"]
-    content {
-      name = "product-code"
-      values = [
-        filter.value
-      ]
-    }
-  }
+#   # Only apply product code filter for some AMI, e.g. debian10
+#   # https://www.terraform.io/docs/language/expressions/dynamic-blocks.html
+#   dynamic "filter" {
+#     for_each = var.amis[var.testing_ami]["ami_product_code"]
+#     content {
+#       name = "product-code"
+#       values = [
+#         filter.value
+#       ]
+#     }
+#   }
 
-  owners = [
-  var.amis[var.testing_ami]["ami_owner"]]
-}
+#   owners = [
+#   var.amis[var.testing_ami]["ami_owner"]]
+# }
```

Comment:
Can we remove this instead of leaving a commented out block?

Reply:
+1. It'll be there in the git history if we need it later.

Reply:
Changed it